src/sentry/tasks/base.py

```diff
@@ -54,7 +54,7 @@ def _wrapped(*args, **kwargs):
     return wrapped
 
 
-def retry(func=None, on=(Exception,), exclude=()):
+def retry(func=None, on=(Exception,), exclude=(), ignore=None):
```

Comment:
@ceorourke I think the default value for `ignore` should be an empty tuple like we have for exclude